src/adapters/wideorbit.js

```diff
@@ -1,213 +1,215 @@
-var bidfactory = require('../bidfactory.js'),
```

Comment:
The line endings in this commit are CRLF (Windows-style), but the existing wideorbit commit uses LF (Unix-style) line endings. Can you keep LF line endings so the diff is easier to read?